other/floydstriangle/floydstriangle.go

```diff
@@ -0,0 +1,24 @@
+// Golang Program to Print Floyd's Triangle.
+/* author Sreejita Roy (https://github.com/pseudonerd16)*/
+
+package main
+
+import "fmt"
+
+func main(){
```

Comment:
Remove the main function and instead create a function.

Reply:
Write a self contained function instead of writing everything in main.